perun-base/src/main/resources/perun-roles.yml

```diff
@@ -4978,6 +4978,28 @@ perun_policies:
     include_policies:
       - default_policy
 
+  getUserExtSourceByUniqueAttributeValue_int_String_policy:
+    policy_roles:
+      - RPC:
```

Comment:
Remove the not-needed roles from these policies, please.

Reply:
Removed